bin/node/runtime/src/lib.rs

```diff
@@ -603,6 +603,8 @@ frame_election_provider_support::generate_solution_type!(
 parameter_types! {
 	pub MaxNominations: u32 = <NposSolution16 as frame_election_provider_support::NposSolution>::LIMIT as u32;
 	pub MaxElectingVoters: u32 = 10_000;
+	pub MaxBackersPerWinner: u32 = u32::max_value();
```

Comment:
Should add a note about the why of this, since it can really raise an eyebrow.

Reply:
The main reason being, that the election provider multi phase pallet (and its miner) is not really designed to respect these bounds. Nonetheless, we can be a bit more gentle here and set `MaxBackersPerWinner` to `MaxElectingVoters`, assuming the near-impossible scenario of all voters backing a single winner, and `MaxWinnersPerPage` to `MaxElectableTargets`, assuming that all targets are present in one page.